addon/-private/execution_context/acceptance.js

```diff
@@ -33,13 +39,21 @@ AcceptanceExecutionContext.prototype = {
     click(selector, container);
   },
 
-  fillIn(selector, container, text) {
-    /* global fillIn */
-    if (container) {
-      fillIn(selector, container, text);
-    } else {
-      fillIn(selector, text);
-    }
+  fillIn(selector, container, options, content) {
+    let $selection = find(selector, container || findClosestValue(this.pageObjectNode, 'testContainer'));
```

Comment:
Note: I changed this from `$element` to `$selection`...seems more accurate, since this could be a collection of multiple elements.